src/libtoast/src/toast_atm.cpp

```diff
@@ -3,9 +3,9 @@
 // All rights reserved.  Use of this source code is governed by
 // a BSD-style license that can be found in the LICENSE file.
 
-// #if !defined(DEBUG)
-// #   define DEBUG
-// #endif
+#if !defined(DEBUG)
```

Comment:
Did you intend to always define "DEBUG"?

Reply:
I meant for all the code in the current DEBUG blocks to be enabled. I changed the preprocessed variable to NO_ATM_CHECKS and the code can be disabled by setting it.